### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by ppbrown** [bolthole/zrep#192] [Announcements]
At long last, the official Zrep version 2.0, that is bash based, for greater portability.

At this point, bash(4.4?) seems to have absorbed almost every single feature I was using, that was previously specific to ksh.
Almost no tweaks were neccessary other than time format, and print -> echo.

Since true ksh is practically impossible to get these days... the future needs be bash based.

I hope everyone, new and old, will continue to get good use out of Zrep

<hr /><em>This discussion was created from the release <a href='https://github.com/bolthole/zrep/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by tschettervictor** [comment]
Great! 

Is this project still going strong?

Are you planning on adding more features and goodies or have you done pretty much all there is?

**Comment by ppbrown** [comment]
I do not have any current plans for large changes.
